pkg/sdk/testint/connections_gen_integration_test.go

```diff
@@ -55,210 +63,187 @@ func TestInt_Connections(t *testing.T) {
 		require.NoError(t, err)
 		t.Cleanup(testClientHelper().Connection.DropFunc(t, id))
 
+		externalObjectIdentifier := sdk.NewExternalObjectIdentifier(accountId, id)
 		assertions.AssertThatObject(t, objectassert.Connection(t, id).
-			HasSnowflakeRegion(region).
-			HasAccountName(accountName).
+			HasSnowflakeRegion(sessionDetails.Region).
+			HasAccountName(sessionDetails.AccountName).
 			HasName(id.Name()).
 			HasComment("test comment for connection").
 			HasIsPrimary(true).
-			HasPrimary(fmt.Sprintf("%s.%s.%s", orgName, accountName, id.Name())).
+			HasPrimaryIdentifier(externalObjectIdentifier.FullyQualifiedName()).
```

Comment:
Change the accepted type to identifier and call FullyQualifiedName inside.

Reply:
wiil do